openbb_terminal/terminal_controller.py

```diff
@@ -93,6 +94,7 @@ class TerminalController(BaseController):
         "exe",
         "guess",
         "news",
+        "news_biztoc",
```

Comment:
Can you call biztoc inside the `news` command and not a separate command? Then users can specify `news --sources Biztoc`